### ISSUE TITLE
Columns Block: Add horizontal alignments

### ISSUE THREAD

**Issue by scruffian** [WordPress/gutenberg#31459]
Since https://github.com/WordPress/gutenberg/pull/13899 we have been able to vertically align the content of a column. It would also be useful to be able to align the content to the left/right. This will help us to create layouts like this:

<img width="871" alt="Screenshot 2021-05-04 at 13 33 07" src="https://user-images.githubusercontent.com/275961/117003836-43ca1880-acdd-11eb-8c27-39576de1cb7f.png">

Of course we could add an alignment to the subscriptions block, but having it at the columns level will mean that all blocks will be able to take advantage of it without needing to implement it themselves.

**Comment by strarsis** [comment]
Yes! Adding (horizontal) alignment to a block to be used inside the column block doesn't really make sense, 
as some blocks don't have something inside that can go to their own edge (content/child blocks).

**Comment by tellthemachines** [comment]
Closing this with the suggestion to instead use a Grid with Stacks inside it:

<img width="1444" height="395" alt="Image" src="https://github.com/user-attachments/assets/8d720a3b-0b13-48cb-abe0-de4096c08959" />

(a Row would also work instead of a Grid if you wish to have differently-sized columns)
